application/modules/clients/models/Mdl_clients.php

```diff
@@ -35,93 +35,97 @@ public function default_order_by()
 
     public function validation_rules()
     {
-        return array(
-            'client_name' => array(
+        return [
+            'client_title' => [
+                'field' => 'client_title',
+                'label' => trans('client_title'),
```

Comment:
@KevinJ30 I removed the "required" part from the validation.
When you enter a new client the "title" isn't required